src/ethereum/constantinople/vm/interpreter.py

```diff
@@ -164,6 +166,51 @@ def process_create_message(message: Message, env: Environment) -> Evm:
     return evm
 
 
+def process_create2_message(message: Message, env: Environment) -> Evm:
+    """
+    Executes a call to create a smart contract via CREATE2 opcode.
```

Comment:
Although I was the one who wrote "smart contract" in `process_create_message`, I'm not sure if it is appropriate for the specs. Can we say something like:
"Executes a call to deploy bytecode via CREATE2 opcode."?